d3a_api_client/resources/api-client-summary-84d221fa-46e1-49e0-823b-26cfb3425a5a.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
I would rename this folder to either `factories` or `fixtures` (and possibly move it inside the `unit_tests/` folder).
I highly recommend using [factory_boy](https://github.com/FactoryBoy/factory_boy) for this. You can check what has already been done in d3a-web: https://github.com/gridsingularity/d3a-web/blob/master/simulation/tests/factories.py

Reply:
Edit: since you're reading these files, using factory boy wouldn't help here.

Reply:
> and possibly move it inside the unit_tests/ folder)

Its deliberately kept here as `unit_tests` is outside `api_client_path`

Reply:
Can you elaborate on this? What's the issue that would be generated if we moved it into `unit_tests/`?

Reply:
The reason I used `api_client_path ` and kept resource files under `d3a_api_client` to have a uniform reference for test being executed locally as well as via `tox` as they both were taking different `cwd`

Reply:
But why not just specifying another path in the tests, without joining it to the `d3a_api_client` module location? These resources are not part of the `d3a_api_client`, they're specifically used in tests (as fixtures) and we should be able to de-couple the two things without breaking any functionality in the client.